lms/djangoapps/ccx/views.py

```diff
@@ -208,9 +213,31 @@ def create_ccx(request, course, ccx=None):
         email_params=email_params,
     )
 
+    assign_coach_role_to_ccx(ccx_id, request.user, course.id)
+
     return redirect(url)
 
 
+def assign_coach_role_to_ccx(ccx_locator, user, master_course_id):
+    """
+    Check if user has ccx_coach role on master course then assign him coach role on ccx only
+    if role is not already assigned. Because of this coach can open dashboard from master course
+    as well as ccx.
+    :param ccx_locator: CCX key
+    :param user: User to whom we want to assign role.
+    :param master_course_id: Master course key
+    """
+    coach_role_on_master_course = CourseCcxCoachRole(master_course_id)
+    # check if user has coach role on master course
+    if coach_role_on_master_course.has_user(user):
+        # Check if user has coach role on ccx.
+        role = CourseCcxCoachRole(ccx_locator)
+        if not role.has_user(user):
+            # assign user role coach on ccx
+            with ccx_course(ccx_locator) as course:
+                allow_access(course, user, "ccx_coach", send_email=False)
+
+
```

Comment:
this is ok @pdpinch because this is a global method. And we put two spaces before and after a global method or vaiable